vars/FreeBSD.yml

```diff
@@ -1,3 +1,11 @@
+__argus_radium_user: argus
+__argus_radium_user_home: /var/log/argus
+__argus_radium_user_shell: /usr/sbin/nologin
+__argus_radium_user_uid: 561
```

Comment:
you could not manage UIDs for regular users without knowing which UIDs are used by the roles in our servers. you mean we should not manage UIDs for regular users as well?